usbredirserver@.service

```diff
@@ -7,7 +7,7 @@ EnvironmentFile=/var/lib/usbredirserver/%i
 
 ExecStart=/usr/sbin/usbredirserver -p %i ${BUS}-${DEV}
 
-ExecStop=/bin/bash -c 'fuser -k ${PORT}/tcp'
```

Comment:
Hi,

Variable is not needed here, because `%1` will be taken from the unit name, eg

```
systemctl start usbredirserver@1122.service
```

will start usbredirserver on prot `1122`

Reply:
That's what I mean.

Reply:
My bad, you're right!